src/StockportWebapp/Views/stockportgov/Directory/Results.cshtml

```diff
@@ -29,15 +29,15 @@
                     <div class="navcard navcard--white navcard--width-100 navcard--no-border">
                         <div class="navcard-content">
                             <div class="navcard-content--padding">
-                                <h3>Filter you results</h3>
+                                <h2 class="h-m">Filter you results</h2>
                                 <p>Choose one or more options to filter your results.</p>
 
                                 @if(Model.Directory.SubDirectories.Any())
                                 {
-                                    <h4>Category</h4>
+                                    <h3 class="h-s">Category</h3>
                                     @foreach(var subDirectory in Model.Directory.SubDirectories)
                                     {
-                                        <p><a href="@Url.Action("DirectoryResults", "Directory", new { slug = subDirectory.Slug })">@subDirectory.Title</a></p>
+                                        <p><a alt="" href="@Url.Action("DirectoryResults", "Directory", new { slug = subDirectory.Slug })">@subDirectory.Title</a></p>
```

Comment:
Do **a** tags need/allow **alt** attributes?

Reply:
This was my mistake - I was fixing and issue in the wrong place for some reason!